content/blog/2016/2016-05-10-jenkins-20-vjam.adoc

```diff
@@ -0,0 +1,355 @@
+open source.
+
+____
+How well is jenkins 2.0 integrate with github for tracking job definitions?
+____
+
+Using the
+link:https://wiki.jenkins-ci.org/display/JENKINS/GitHub+Organization+Folder+Pluginp[GitHub
+Organization Folder plugin], Jenkins can automatically detect a `Jenkinsfile` in
+source repositories to create Pipeline projects.
+
+
+____
+Please make the ability for re-run failed stages Open Source too :)
+____
+
+This has been passed on to our friends at CloudBees for consideration :)
+
+____
+If `Jenkinsfile` is in the repo, co-located with code, does this mean Jenkins can
+auto-detect new jobs for different branches?
+____
+
+This is possible using the
+link:/doc/pipeline/#creating-multibranch-pipelines[Pipeline Multibranch plugin].
+
+
+____
+What documentation sources are there for Pipeline?
+____
+
+Our link:/doc/[documentation section] contains a number of pagesaround Pipeline.
+There is also additional documentation and examples in the plugin's
+link:https://github.com/jenkinsci/pipeline-plugin[git repository] and the
+link:https://github.com/jenkinsci/pipeline-examples[jenkinsci/pipeline-examples]
+repository. (contributions welcome!)
+
+____
+Where we can find the DSL method documentation?
+____
+
+There is generated link:/doc/pipeline/steps[documentation on jenkins.io] which
+incldues steps from all public plugins. Inside of a running Jenkins instance,
+you can also navigate to
+link:http://localhost:8080/workflow-cps-snippetizer/dslReference[localhost:8080/workflow-cps-snippetizer/dslReference]
+to see the documentation for the plugins which are installed in that instance.
+
+
+____
+If Pipeline is not support some plugins (there is a lot actually), i needed
+SonarQube Runner but unfortunately it's not supported yet, in Job DSL plugin i
+can use "Configure Block" and cover any plugin via XML, how i can achieve the
+same with a Pipeline?
+____
+
+Not at this time
+
+____
+Is there a possibility to create custom tooltips i.e. with a quick reference or
+a link to internal project documentation? Might be useful i.e. for junior team
+members who need to refer to external docs.
+____
+
+Not generally. Though in the case of Pipeline global libraries, you can create
+descriptions of vars/functions like `standardBuild` in the demo, and these will
+appear in Snippet Generator under Global Variables.
+
+____
+Oh pipeline supports joining jobs? It's really good, but I cannot find document
+at https://jenkins.io/doc/ could you tell me where is it?
+____
+
+There is a `build` step, but the Pipeline system is optimized for single-job
+pipelines
+
+____
+We have multiple projects that we would like to follow the same pipeline.  How
+would I write a common pipeline that can be shared across multiple projects.
+____
+
+You may want to look at implementing some additional steps using the
+link:https://github.com/jenkinsci/pipeline-examples/tree/master/global-library-examples[Global
```

Comment:
Please link me to where this "full name" is being used, because it certainly isn't in the link.